### PR TITLE
set DeleteNs true to delete cms

### PR DESCRIPTION
**Pull request by chengjoey**
#### What type of this PR

Add one of the following kinds:
/kind bug

#### What this PR does / why we need it:
set DeleteNs true to delete cms

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda-org.erda.cloud/erda/dop/projects/387/issues/all?id=204922&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIl0sImFzc2lnbmVlSURzIjpbIjEwMDEyMDUiXX0%3D&issueTable__urlQuery=eyJwYWdlTm8iOjF9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=431&type=BUG)

### CODE REVIEW

File: modules/dop/services/autotest/global_config.go [hunk 1]
```diff
@@ -307,7 +307,7 @@ func (svc *Service) DeleteGlobalConfig(req apistructs.AutoTestGlobalConfigDelete
 	if _, err := svc.cms.DeleteCmsNsConfigs(utils.WithInternalClientContext(context.Background()), &cmspb.CmsNsConfigsDeleteRequest{
 		Ns:             req.PipelineCmsNs,
 		PipelineSource: apistructs.PipelineSourceAutoTest.String(),
-		DeleteNs:       false,
+		DeleteNs:       true,
```
Comment (Effet) [inline modules/dop/services/autotest/global_config.go:310]: how this bug introduced? just forget to set `true`?

Comment (chengjoey) [reply modules/dop/services/autotest/global_config.go:310]: Parameter ‘deleteNS' is true in version release/1.1,but it was be set as false after pb refector

**Review by Effet — APPROVED** [review]

**Pull request merged**